docs/tutorials/mask_maps.ipynb

```diff
@@ -15,19 +15,27 @@
     "\n",
     "### Context\n",
     "\n",
+    "#### Fit masks\n",
+    "\n",
+    "The region of interest used for the fit can defined through the dataset `mask_fit` attribute.\n",
+    "The `mask_fit` is a map containing boolean values where pixels used in the fit are stored as True.  \n",
+    "\n",
+    "2D and 3D analyses usually require to work with a wider map than the region of interrest so sources laying outside but reconstructed inside because of the PSF are correcly taken into account. Then the `mask_fit` have to include a margin that take into accout the PSF width. We will show an example in the boundary mask sub-section.\n",
+    "\n",
+    "The `mask_fit` also can be used to exclude sources or confude regions for which we don't have good enough models. In that case the masking is an extra security, it is prefereable to include the available models even if the sources are masked and frozen.\n",
+    "\n",
+    "Note that a dataset contains also a `mask_safe` attribute that is not meant to be modified directly by users. The  `mask_safe` defined only from the cuts used in the data reductions (more details [here](https://docs.gammapy.org/dev/makers/index.html#safe-data-range-handling)).\n",
+    "\n",
+    "#### Exclusion masks\n",
+    "\n",
     "Background templates stored in the DL3 IRF are often not reliable enough to be used without some corrections. A set of common techniques to perform background or normalisation from the data is implemented in gammapy: reflected regions for 1D spectrum analysis, field-of-view (FoV) background or ring background for 2D and 3D analyses.\n",
     "\n",
     "To avoid contamination of the background estimate from gamma-ray bright regions these methods require to exclude those regions from the data used for the estimation. To do so, we use exclusion masks. They are maps containing boolean values where excluded pixels are stored as False.  \n",
     "\n",
-    "**Objective: Build an exclusion mask around the Crab nebula excluding gamma-ray sources in the region.**\n",
     "\n",
-    "### Proposed approach\n",
     "\n",
-    "Here we have to build a `Map` object, we must first define its geometry and then we can determine which pixels to exclude.\n",
+    "**Objective: Build a mask around the Crab nebula excluding gamma-ray sources in the region.**\n",
```

Comment:
Listing the of the demonstrated functionalities at the very beginning of the notebook (rather than having to scroll through the entire notebook to know if something exists) can be quite useful.
Something like:

Here we show  :
 - Creating masks from scratch 
 - Combining multiple masks
 - Extending and reducing an existing mask
 - Serialisation of masks